### ISSUE TITLE
ACNL - Welcome amiibo - amiibo Camera never finishes loading

### ISSUE THREAD

**Issue by ghost** [citra-emu/citra#3438]
Commit: 3c03da1
![freeze](https://user-images.githubusercontent.com/22228082/36239609-a2d6f510-11d0-11e8-8f3e-3d7f277f2fdd.PNG)
[Log](https://github.com/citra-emu/citra/files/1726359/log.txt)

**Comment by BynariStar** [comment]
Does this still happen now with #3566 and #3764 merged?

**Comment by zhaowenlan1779** [comment]
Looks more related with these lines from the log
```
[  15.430045] Service.FS <Error> core/file_sys/savedata_archive.cpp:OpenDirectory:313: Path not found C:\Users\79869\Desktop\nigthly-mingw/user/sdmc/Nintendo 3DS/00000000000000000000000000000000/00000000000000000000000000000000/title/00040000/00198f00/data/00000001/amiibo.dat
[  15.430060] Service.FS <Error> core/hle/service/fs/fs_user.cpp:OpenDirectory:431: failed to get a handle for directory type=4 size=24 data=[Wchar: /amiibo.dat]
[  15.430266] Service.FS <Error> core/file_sys/savedata_archive.cpp:OpenFile:53: Non-existing file C:\Users\79869\Desktop\nigthly-mingw/user/sdmc/Nintendo 3DS/00000000000000000000000000000000/00000000000000000000000000000000/title/00040000/00198f00/data/00000001/amiibo.dat can't be open without mode create.
[  15.430276] Service.FS <Error> core/hle/service/fs/fs_user.cpp:OpenFile:91: failed to get a handle for file [Wchar: /amiibo.dat]
```

@BynariStar I suppose so, as camera integration does not really change much about Service.CAM.

**Comment by zhaowenlan1779** [comment]
Fixed by #5116
